var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -129,13 +129,15 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     variant("aws", default=False, description="Build with Amazon AWS Platform support")
```

Comment:
I think MKL should also default to True when not on macOS

Reply:
Is that based on `.bazelrc` or something else? I see lines like `build:mkl --define=build_with_mkl=true --define=enable_mkl=true`, but I think `build:NAME` sections are only activated when `--config=NAME` is used. Which it might be, but I can't tell one way or the other.

I also think turning it on would be fine the conflicts for `+mkl` are all present (is it just `darwin`?) and it's not going to fail/be slower on non-Intel platforms.

Reply:
Oh yeah, that was based on those sections. I don't know bazel 😄